manager/ui/hawtio/plugins/api-manager/html/forms/edit-gateway.html

```diff
@@ -108,7 +108,11 @@ <h2 class="title" apiman-i18n-key="edit-gateway">Edit Gateway</h2>
       <!-- Update Button -->
       <div class="row">
         <button id="update-gateway" ng-disabled="!isValid || !isDirty" apiman-action-btn="" class="btn btn-primary" data-field="updateButton" apiman-i18n-key="update-gateway" placeholder="Updating..." data-icon="fa-cog" ng-click="updateGateway()">Update Gateway</button>
-        <a id="cancel" href="javascript:window.history.back()" class="btn btn-default btn-cancel" data-field="cancelButton" apiman-i18n-key="cancel">Cancel</a>
+        <a id="cancel"
+           ng-click="cancel();"
+           class="btn btn-default btn-cancel cancel"
+           apiman-i18n-key="cancel">Cancel
+        </a>
```

Comment:
If you put the </a> end-tag on a separate line it will introduce whitespace in the text.  Can you move it up?